packages/lightning-language-server/package.yaml

```
@@ -0,0 +1,17 @@
+
+name: lwc-language-server
+description: Language Server for Lightning Web Components.
+homepage: https://github.com/forcedotcom/lightning-language-server/
+licenses:
+  - BSD-3-Clause
+languages:
+  - HTML
+  - JavaScript
+categories:
+  - LSP
+source:
+  id: pkg:npm/@salesforce/lwc-language-server@v4.7.3
```

The `@` will need to be percent-encoded as otherwise anything right of it is parsed as the version:

```suggestion
  id: pkg:npm/%40salesforce/lwc-language-server@v4.7.3
```